solr/solrj/src/java/org/apache/solr/client/solrj/SolrRequest.java

```diff
@@ -132,6 +138,10 @@ public String getPath() {
     return path;
   }
 
+  /**
+   * The path of this request following the core or collection (if applicable). Sometimes called the
```

Comment:
[0] Maybe tweak the wording here to make it a little clearer what value this method expects for requests that **don't** target a particular core/collection.

Perhaps something like:

> The HTTP path suffix used when sending this request.
>
> Commonly used with {@link #getBasePath()} and {@link #getCollection()} to assemble the full HTTP path for the request.
> 
> For requests targeting a specific core or collection, callers need only specify those path components that follow the collection name (e.g. "/select" for the request "/solr/coreName/select"). This is sometimes called the "handler" path.
> 
> For requests that don't target a particular core or collection, callers should specify any path components that follow "/solr" or "/api" (e.g. "/admin/info/health" in "/solr/admin/info/health").

Feel free to ignore this advice though - any javadocs here are an improvement.